docs/_deployments/aws-lambda.md

```diff
@@ -140,12 +140,7 @@ const awsLambdaReceiver = new AwsLambdaReceiver({
 // Initializes your app with your bot token and the AWS Lambda ready receiver
 const app = new App({
     token: process.env.SLACK_BOT_TOKEN,
-    receiver: awsLambdaReceiver,
-    // The `processBeforeResponse` option is required for all FaaS environments.
```

Comment:
Instead of deleting this, we want to say "If you go with `AwsLambdaReceiver` like this example does, you can omit `processBeforeResponse: true`. But, if you use other receivers such as ExpressReceiver for OAuth flow support, `processBeforeResponse: true` is required."

See also:
* https://github.com/slackapi/bolt-js/issues/1062
* https://github.com/slackapi/bolt-js/issues/1028
* https://github.com/slackapi/bolt-js/pull/1066/files#diff-2130e8227ce7a48a4f16812225568f8c3bca634968bd620e3c47fe9fa5f692c3R130

Reply:
Sounds good, I'll rework this then.